internal/runners/packages/install.go

```diff
@@ -6,44 +6,82 @@ import (
 	"github.com/ActiveState/cli/internal/logging"
 	"github.com/ActiveState/cli/internal/rtutils/ptr"
 	"github.com/ActiveState/cli/internal/runbits/runtime/requirements"
+	"github.com/ActiveState/cli/pkg/localcommit"
 	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/types"
 	"github.com/ActiveState/cli/pkg/platform/model"
 )
 
 // InstallRunParams tracks the info required for running Install.
 type InstallRunParams struct {
-	Packages  captain.PackagesValue
-	Timestamp captain.TimeValue
-	Revision  captain.IntValue
+	Packages      captain.PackagesValue
+	Timestamp     captain.TimeValue
+	Revision      captain.IntValue
+	NamespaceType *model.NamespaceType
 }
 
 // Install manages the installing execution context.
 type Install struct {
 	prime primeable
 }
 
+type errNamespaceMismatch struct {
```

Comment:
See https://github.com/ActiveState/cli/pull/3402/files#diff-9660d67939c62dc12e53e50721067074f16c958fd6f5f554b86d1b8de83be503R80 for why this exists.